com.hazelcast.jet.core.SplitBrainTest.when_newMemberIsAddedAfterClusterSizeFallsBelowQuorumSize_then_jobRestartDoesNotSucceed

https://hazelcast-l337.ci.cloudbees.com/job/jet-pr-builder/com.hazelcast.jet$hazelcast-jet-core/2834/testReport/junit/com.hazelcast.jet.core/SplitBrainTest/when_newMemberIsAddedAfterClusterSizeFallsBelowQuorumSize_then_jobRestartDoesNotSucceed/

```
Error Message
expected:<RESTARTING> but was:<FAILED>
Stacktrace
java.lang.AssertionError: expected:<RESTARTING> but was:<FAILED>
	at com.hazelcast.jet.core.SplitBrainTest.lambda$when_newMemberIsAddedAfterClusterSizeFallsBelowQuorumSize_then_jobRestartDoesNotSucceed$26(SplitBrainTest.java:304)
	at com.hazelcast.jet.core.SplitBrainTest.when_newMemberIsAddedAfterClusterSizeFallsBelowQuorumSize_then_jobRestartDoesNotSucceed(SplitBrainTest.java:304)
```

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/SplitBrainTest.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/SplitBrainTest.java
@@ -296,12 +296,12 @@ public class SplitBrainTest extends JetSplitBrainTestSupport {
         assertOpenEventually(StuckProcessor.executionStarted);
 
         for (int i = 1; i < clusterSize; i++) {
-            instances[i].getHazelcastInstance().getLifecycleService().terminate();
+            instances[i].shutdown();
         }
 
         StuckProcessor.proceedLatch.countDown();
 
-        assertTrueEventually(() -> assertEquals(RESTARTING, job.getStatus()));
+        assertTrueEventually(() -> assertEquals(RESTARTING, job.getStatus()), 10);
 
         createJetMember(jetConfig);
 

